dace/transformation/dataflow/map_distribution.py

```diff
@@ -172,7 +160,7 @@ def _prod(sequence):
 
             elif isinstance(desc, data.Array):
 
-                local_name, local_arr = sdfg.add_temp_transient([(desc.total_size) // sz],
+                local_name, local_arr = sdfg.add_temp_transient([sympy.floor(desc.total_size / sz)],
```

Comment:
In the past, we had issues using sympy's floor/ceiling methods. We probably don't have tests that check cases where this might be an issue (i.e., when the numbers don't divide evenly). This is something to keep in mind for future reference.

Reply:
This started raising exceptions for some reason with Integer-class `total_size`. I also don't know why it started raising now.